core/che-core-api-dto/src/main/java/org/eclipse/che/dto/server/DtoFactory.java

```diff
@@ -386,8 +386,13 @@ public Gson getGson() {
     private <T> DtoProvider<T> getDtoProvider(Class<T> dtoInterface) {
         DtoProvider<?> dtoProvider = dtoInterface2Providers.get(dtoInterface);
         if (dtoProvider == null) {
-            throw new IllegalArgumentException("Unknown DTO type " + dtoInterface);
+            if (dtoInterface.isAnnotationPresent(DTO.class)) {
+                throw new IllegalArgumentException("Implementation for DTO type " + dtoInterface + " isn't found.");
```

Comment:
Thank you for advice.
The main reason of this request is to mention **implementation** in some way in error message.
But in the method which throws exception we operate with term **provider**.
Next message is enclosing both of reasons:
> Provider of implementation for DTO type ...